src/settingsmanager.h

```diff
@@ -67,9 +74,11 @@ public slots:
     QString m_downloadDir;
     QString m_monitorDir;
     bool m_moveToTrash;
+    bool m_reopenTab;
     QList<QVariant> m_langList;
     QList<QVariant> m_categoryList;
     QList<QVariant> m_contentTypeList;
+    QStringList m_tabList;
```

Comment:
I don't think that the list of open tabs belongs to the app settings. It should rather be considered part of the app session.

Under Linux the settings are stored in `~/.config/Kiwix/Kiwix-desktop.conf`.  Other information is written under `~/.local/share/kiwix/`. There is a `kiwix.session` file in that latter directory that seems to always be empty. I think that the list of tabs should be written into that file.

@kelson42 Does it make sense?

Reply:
Agrees with moving them from settings to application. I believe the QSessionManager is the one that writes the session file? On the documentation it still mentions to use QSettings to store those session information. I can probably move QSettings code to work with QSessionManager.

Reply:
So I found kiwix.session is located in the library data directory. I can just use QSettings to write to this file if we confirm this file isn't used is any other placed.

Reply:
LGTM

Reply:
Ok, we can't do that. kiwix.session is used by aria2 to restore download sessions when the application closes. I can put the content in another file in the data directory.

Reply:
Done